book/label.rb

```diff
@@ -0,0 +1,17 @@
+require_relative 'book'
+
+class Label
+  attr_accessor :title, :color, :items
+
+  def initialize(title, color)
+    @id = Random.rand(1...1000)
+    @title = title
+    @color = color
+    @items = []
+  end
+
```

Comment:
- [ ] Also please add the requirement to 4 people the same as the music album `the genre, and source` are not added, please add them.

![Capture web_1-7-2022_122952_gitlab com](https://user-images.githubusercontent.com/61562626/176886921-e49ceed5-c224-4460-9dc2-f2fcb0c70a79.jpeg)